chain/chunks/src/test_utils.rs

```diff
@@ -130,3 +130,159 @@ impl SealsManagerTestFixture {
+                        mock_shard_id,
+                        false,
+                    );
+                    tracks_shard
+                }
+            })
+            .cloned()
+            .unwrap();
+        let mock_chunk_part_owner = validators
+            .into_iter()
+            .flatten()
+            .find(|v| v != &mock_chunk_producer && v != &mock_shard_tracker)
+            .unwrap();
+
+        let mut producer_shard_manager = ShardsManager::new(
+            Some(mock_chunk_producer.clone()),
+            mock_runtime.clone(),
+            mock_network.clone(),
+        );
+        let receipts = Vec::new();
+        let receipts_hashes = mock_runtime.build_receipts_hashes(&receipts);
+        let (receipts_root, _) = merkle::merklize(&receipts_hashes);
+        let (mock_chunk, mock_merkles) = producer_shard_manager
+            .create_encoded_shard_chunk(
+                mock_parent_hash,
+                Default::default(),
+                Default::default(),
+                mock_height,
+                mock_shard_id,
+                0,
+                1000,
+                0,
+                Vec::new(),
+                Vec::new(),
+                &receipts,
+                receipts_root,
+                MerkleHash::default(),
+                &signer,
+                &mut rs,
+            )
+            .unwrap();
+
+        let all_part_ords: Vec<u64> =
+            (0..mock_chunk.content.parts.len()).map(|p| p as u64).collect();
+        let mock_part_ords = all_part_ords
+            .iter()
+            .copied()
+            .filter(|p| {
+                mock_runtime.get_part_owner(&mock_parent_hash, *p).unwrap() == mock_chunk_part_owner
+            })
+            .collect();
+        let encoded_chunk = mock_chunk.create_partial_encoded_chunk(
+            all_part_ords.clone(),
+            Vec::new(),
+            &mock_merkles,
+        );
+        let chain_store = ChainStore::new(store, 0);
+
+        ChunkForwardingTestFixture {
+            mock_runtime,
+            mock_network,
+            chain_store,
+            mock_part_ords,
+            mock_chunk_part_owner,
+            mock_shard_tracker,
+            mock_chunk_header: encoded_chunk.header,
+            mock_chunk_parts: encoded_chunk.parts,
+            rs,
+        }
+    }
+}
+
+impl ChunkForwardingTestFixture {
+    pub fn make_partial_encoded_chunk(&self, part_ords: &[u64]) -> PartialEncodedChunk {
+        let parts = part_ords
+            .iter()
+            .copied()
+            .flat_map(|ord| self.mock_chunk_parts.iter().find(|part| part.part_ord == ord))
+            .cloned()
+            .collect();
+        PartialEncodedChunk { header: self.mock_chunk_header.clone(), parts, receipts: Vec::new() }
+    }
+}
+
+fn make_validators(n: usize) -> Vec<Vec<AccountId>> {
+    if n > 26 {
+        panic!("I can't make that many validators!");
+    }
+
+    let letters = (b'A'..=b'Z')
```

Comment:
capital letters are not allowed in account ids. We don't check it here but it's better to be consistent.